aws/resource_aws_codebuild_webhook_test.go

```diff
@@ -0,0 +1,78 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/codebuild"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAwsCodeBuildWebhook_basic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAwsCodeBuildWebhookDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCodeBuildWebhookConfig_basic(acctest.RandString(5)),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAwsCodeBuildWebhookExists("aws_codebuild_webhook.test"),
+					resource.TestCheckResourceAttrSet("aws_codebuild_webhook.test", "url"),
```

Comment:
We should probably be a little more cautious and at least partially validate the value as `resource.TestCheckResourceAttrSet()` will pass with empty strings and I believe even `d.Set("XXX", nil)`

e.g.

```go
resource.TestMatchResourceAttr("aws_codebuild_webhook.test", "url", regexp.MustCompile(`^https://`))
```